### DISCUSSION TITLE
Create a collection package

### DISCUSSION THREAD

**Discussion by damooo** [adobe/spectrum-web-components#3941] [Ideas]
It is difficult to install and manage each and every component as a package. It would be great, if a collection package is published, that re-exports components from different sub modules. This way bundlers also can treeshake, and one package is enough to install.

**Comment by Westbrook** [comment]
We're discussing the possibilities of this in #2506. However, we're not likely to be addressing this in the near future. The packages are distributed with fixed versions, so as long as you include dependencies in a matched version you'll receive what amounts to a "collection package".
